Denis_Ng/w2_d5/main.js

```diff
@@ -0,0 +1,22 @@
+
+// (1)
+var heading = document.getElementById('heading');
```

Comment:
Dennis! hows it going?
for question 1 you need to use -> document.getElementsByTagName('h1')[1];